test/integration/default/serverspec/windows_spec.rb

```diff
@@ -1,26 +0,0 @@
-# these tests only for windows
-require 'spec_helper'
-
-expected_rules = [
```

Comment:
In a comment on Feb 4, we concluded to make a seperate fixture test cookbook recipe for windows, and at that time, I also did this change to seperate test suites for windows and others so that test suites will run on its own.. And you seemed to happy with this on your comment on Feb 11. What was changed since then?